website/static/main.css

```diff
@@ -72,6 +72,33 @@ pre {
 .header-anchor { font-size: 100%; visibility: hidden;}
 h1:hover a, h2:hover a, h3:hover a, h4:hover a { visibility: visible}
 
+/* seaarchBox */
```

Comment:
It's just a nit but the `searchBox` is not vertically aligned right now like the rest of the links. You can add some simple CSS to fix this
```css
.searchBox {
  height: 100%;
  display: flex;
  align-items: center;
}
```

Reply:
Alright. I would proceed with the changes now. 
Thanks for the suggestions @prmsrswt